### PR TITLE
Feature/captcha

### PR DESCRIPTION
**Pull request by btry**
captcha implementation for sign up

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,22 +12,23 @@
...
-	"repositories" : [{
-			"type" : "vcs",
-			"url" : "https://github.com/flyve-mdm/flyve-mdm-glpi-test"
-		}
-	]
-}
\ No newline at end of file
+    },
+    "repositories" : [{
+            "type" : "vcs",
```
Comment (ajsb85) [inline composer.json:30]: Type: git

File: composer.json [hunk 2]
```diff
@@ -12,22 +12,23 @@
...
+            "url" : "https://github.com/flyve-mdm/flyve-mdm-glpi-test"
```
Comment (ajsb85) [inline composer.json:31]: "url" : "https://github.com/flyve-mdm/flyve-mdm-glpi-test.git"

File: composer.json (hunk 1 above)
Comment (btry) [reply composer.json:30]: no, the specs says vcs. For git, svn, mercurual ...

File: install/installer.class.php [hunk 3]
```diff
@@ -180,9 +181,9 @@ protected function createServiceProfileAccess() {
    protected function createInactiveRegisteredProfileAccess() {
       // create profile for registered users
       $profileId = self::getOrCreateProfile(
-            __("Flyve MDM inactive registered users", "flyvemdmdemo"),
-            __("inactive registered FlyveMDM users. Created by Flyve MDM - do NOT modify this comment.", "flyvemdmdemo")
-            );
+         __("Flyve MDM inactive registered users", "flyvemdmdemo"),
+         __("inactive registered FlyveMDM users. Created by Flyve MDM - do NOT modify this comment.", "flyvemdmdemo")
```
Comment (ajsb85) [inline install/installer.class.php]: Flyve MDM witht the space " "

File: composer.json (hunk 2 above)
Comment (btry) [reply composer.json:31]: .git is not necessary to clone the repo, and doing this way makes easier for someone to pick the url and browse to the project.

File: install/installer.class.php (hunk 3 above)
Comment (btry) [reply install/installer.class.php]: +1

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Comment by btry** [comment]
needs an update not pushed in the test tools package

**Pull request merged**